packages/swapper/src/swappers/utils/constants.ts

```diff
@@ -1,2 +1,3 @@
 export const APPROVAL_GAS_LIMIT = '100000' // Most approvals are around 40k, we've seen 72k in the wild, so 100000 for safety.
 export const DEFAULT_SLIPPAGE = '0.03' // 3%
+export const ALLOWABLE_MARKET_MOVEMENT = '0.005' // 0.5% is what https://app.thorswap.finance/ uses
```

Comment:
Now that our THORChain slippage percentage values are so tight (accurate) we need to account for market movements.